src/locales/he/translation.json

```diff
@@ -73,16 +72,16 @@
   "Collapse": "הקטנה",
   "Expand": "הגדלה",
   "Add Layer": "הוספת שכבה",
-  "Zoom:": "זום:",
   "Search": "חיפוש",
+  "Zoom:": "זום:",
```

Comment:
Why was this moved? I would consider sorting the files alphabetically - keywise...

Reply:
The code generation `npm run i18n:refresh` orders the entries like this.

Reply:
Yeah exactly. Idk... I would kinda expect consistent ordering across machines but maybe it doesn't. I just ran the refresh command; no manual edits.